guides/common/modules/proc_configuring-host-based-authentication-control.adoc

```diff
@@ -1,62 +1,92 @@
 ----
 # klist
 ----
-. Create HBAC service and rule on the {FreeIPA} server and link them together.
-The following examples use the PAM service name _{project-context}-prod_.
-Execute the following commands on the {FreeIPA} server:
+. Create a new PAM service:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# ipa hbacsvc-add {project-context}-prod
-# ipa hbacrule-add allow_{project-context}_prod
-# ipa hbacrule-add-service allow_{project-context}_prod --hbacsvcs={project-context}-prod
+# ipa hbacsvc-add _{project-context}-prod_
 ----
-. Add the user who is to have access to the service {project-context}-prod, and the hostname of {ProjectServer}:
+. Create a new HBAC rule:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# ipa hbacrule-add-user allow_{project-context}_prod --user=_username_
-# ipa hbacrule-add-host allow_{project-context}_prod --hosts=_{foreman-example-com}_
+# ipa hbacrule-add _allow-{project-context}-prod_
 ----
+. Add the PAM service to the HBAC rule:
 +
-Alternatively, host groups and user groups can be added to the _allow_{project-context}_prod_ rule.
-. To check the status of the rule, execute:
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# ipa hbacrule-add-service _allow-{project-context}-prod_ --hbacsvcs=_{project-context}-prod_
+----
+. Add the user who you want to be able to access to {Project} and the {ProjectServer} hostname to the HBAC rule:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# ipa hbacrule-find {project-context}-prod
-# ipa hbactest --user=_username_ --host=_{foreman-example-com}_ --service={project-context}-prod
+# ipa hbacrule-add-user _allow-{project-context}-prod_ --user=_ipa-user_
+# ipa hbacrule-add-host _allow-{project-context}-prod_ --hosts=_{foreman-example-com}_
+----
++
+You can also add {FreeIPA} host groups and user groups to the HBAC rule.
+. Optional: Verify the status of the rule:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-. Ensure the allow_all rule is disabled on the {FreeIPA} server.
+# ipa hbacrule-find _{project-context}-prod_
+# ipa hbactest --user=_ipa-user_ --host=_{foreman-example-com}_ --service=_{project-context}-prod_
+----
+. Disable the default `allow_all` rule:
 ifndef::orcharhino[]
 For instructions on how to do so without disrupting other services see the https://access.redhat.com/solutions/67895[How to configure HBAC rules in IdM] article on the Red{nbsp}Hat Customer Portal.
 endif::[]
-. Configure the {FreeIPA} integration with {ProjectServer} as described in xref:Configuring_FreeIPA_Authentication_on_Server_{context}[].
-On {ProjectServer}, define the PAM service as root:
-+
+
+
+On {ProjectServer}, a {Project} administrator re-runs {foreman-installer} to define the _{project-context}-prod_ PAM service:
+
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
 # {foreman-installer} --foreman-pam-service={project-context}-prod
```

Comment:
As noted above, I wonder if this is correct. On line 46 you write "Create a new PAM service" which to me implies something writes out a file in `/etc/pam.d` that the installer will then overwrite.

Reply:
I rewrote this part to use FreeIPA terminology (I dropped the reference to the PAM service; in the context of a procedure to be performed on the FreeIPA server, it's more appropriate to call it a HBAC service). I'm also now using _italics_ only for the values that can actually be replaced by users, not for values like `{project-context}-prod` and `allow_all`.